.github/workflows/ci_clippy.yml

```diff
@@ -9,3 +9,6 @@ jobs:
 
       - name: Run clippy
         run: cargo clippy -- -D warnings
+      
+      - name: No default features
+        run: cargo clippy --no-default-features -- -D warnings
```

Comment:
will clippy catch any build error?

Reply:
Yes. `-- -D warnings` is passing deny warnings to `rustc`.

Reply:
Notably, the warning I fix in this PR is produced with `cargo check --no-default-features`, as well as `cargo clippy --no-default-features`.